src/EnergyPlus/HVACManager.cc

```diff
@@ -2725,6 +2730,46 @@ namespace HVACManager {
 
 	}
 
+	void
+	CheckMassBalances()
+	{
+		using namespace Fans;
+		using namespace Pumps;
+		using General::RoundSigDigits;
+		using DataHVACGlobals::SmallMassFlow;  // Air
+		using DataBranchAirLoopPlant::MassFlowTolerance;  // Liquid
+
+		for (int FanNum = 1; FanNum <= NumFans; ++FanNum) {
+
+//			if (abs(Node( Fan( FanNum ).InletNodeNum ).MassFlowRate - Fan( FanNum ).InletAirMassFlowRate) > SmallMassFlow) {
+//				ShowFatalError(Fan(FanNum).FanType + ", Name: " + Fan(FanNum).FanName + ": Fan inlet mass flow rate (" + RoundSigDigits(Fan( FanNum ).InletAirMassFlowRate, 4) + ") does not match inlet node mass flow rate (" + RoundSigDigits(Node( Fan( FanNum ).InletNodeNum ).MassFlowRate, 4) + ").");
+//			}
+//
+//			if (abs(Node( Fan( FanNum ).OutletNodeNum ).MassFlowRate - Fan( FanNum ).OutletAirMassFlowRate) > SmallMassFlow) {
+//				ShowFatalError(Fan(FanNum).FanType + ", Name: " + Fan(FanNum).FanName + ": Fan outlet mass flow rate (" + RoundSigDigits(Fan( FanNum ).OutletAirMassFlowRate, 4) + ") does not match outlet node mass flow rate (" + RoundSigDigits(Node( Fan( FanNum ).OutletNodeNum ).MassFlowRate, 4) + ").");
+//			}
+
+			if (abs(Node( Fan( FanNum ).InletNodeNum ).MassFlowRate - Node( Fan( FanNum ).OutletNodeNum ).MassFlowRate) > SmallMassFlow) {
+//				ShowFatalError(Fan(FanNum).FanType + ", Name: " + Fan(FanNum).FanName + ": Fan outlet mass flow rate (" + RoundSigDigits(Node( Fan( FanNum ).OutletNodeNum ).MassFlowRate, 4) + ") does not match fan inlet node mass flow rate (" + RoundSigDigits(Node( Fan( FanNum ).InletNodeNum ).MassFlowRate, 4) + ").");
+// can we get a handle on why this is happening before we start crashing user files?
```

Comment:
Sorry, yes, that's what I was trying to say.  And yes, in this case, my understanding is that they should be in synch.  Unless cycling fans do something different with the Fan object mass flow rate vs the node mass flow rate.  @rraustad ?